src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -72,7 +73,12 @@ public ReadOnlyAddressBook getAddressBook() {
 
     @Override
     public ObservableList<Person> getFilteredStudentList() {
-        return model.getFilteredPersonStudentList();
+        return model.getFilteredStudentList();
+    }
+
+    @Override
+    public ObservableList<? extends Displayable> getFilteredAssessmentList() {
```

Comment:
Will the code still work if we use "ObservableList<Assessment>"?

Reply:
yep! I will change it, thanks for pointing this out :D